webapp/src/mutator.ts

```diff
@@ -205,7 +205,7 @@ class Mutator {
         }
 
         const newTemplate = template || {
-            id: Utils.createGuid(),
+            id: Utils.createGuid(''),
```

Comment:
same here, I think this ends up being a `block`.

Reply:
Fixed via https://github.com/mattermost/focalboard/pull/1369/commits/03daaf722a0bff201708cf27b0434c8fbfef4f4c